Plots/Bar/NCL_bar_1.py

```diff
@@ -0,0 +1,166 @@
+"""
+NCL_bar_1.py
+============
+This script illustrates the following concepts:
+   - Drawing bars instead of curves in an XY plot
+   - Changing the aspect ratio of a bar plot
+   - Drawing bars up or down based on a Y reference value
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/bar_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/bar_1_1_lg.png, https://www.ncl.ucar.edu/Applications/Images/bar_1_2_lg.png
+                         and https://www.ncl.ucar.edu/Applications/Images/bar_1_3_lg.png
```

Comment:
```suggestion
    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/bar_1_1_lg.png, https://www.ncl.ucar.edu/Applications/Images/bar_1_2_lg.png and https://www.ncl.ucar.edu/Applications/Images/bar_1_3_lg.png
```

While it's ugly to have one really long line with a bunch of urls, the docs don't generate correctly if a line break is added:

![image](https://user-images.githubusercontent.com/38434768/121752179-d45dfa80-cacc-11eb-8462-02679f9fd309.png)